control_node/config/base_configs.yaml

```diff
@@ -0,0 +1,13 @@
+main_node: 
+  ros__parameters: 
+    tracking_laser: [0.2, 0.0, 0.0]
+    burn_color: [0.0, 0.0, 1.0]
+    burn_time: 5
+camera_node:
+  ros_parameters: 
+    video_dir: '/opt/video_stream'
+    debug_video_dir: '/opt/debug_video_stream'
+    save_debug: True
+    frame_period: .1
+    rgb_size: [1920, 1080]
```

Comment:
there are some sources that say that the optimal depth resolution for the D435 is 848x480. Using a resolution higher than 848x480 would give worse depth quality.
https://community.intel.com/t5/Items-with-no-label/D435-Resolution-VS-Depth-Image-Quality/td-p/548105
https://dev.intelrealsense.com/docs/tuning-depth-cameras-for-best-performance

Reply:
I’m down, I’ll change it